### ISSUE TITLE
évaluation sur le test_data

### ISSUE THREAD

**Issue by nKachout** [MLoum/LifetimeFitNeuralNetwork#1]
le réseau nous donne toujours un nombre de tau correct sur le test_data égale à 0 et je ne comprends pas encore pourquoi même après avoir utilisé plusieurs valeur différentes d’hyper-paramètres, néanmoins j'ai utilisé le network2 et d'après le terminal le coût change bien en fonction des hyper-paramètres mais le coût reste important j'ai réussi à le réduire à 10 au maximum mais toujours 0 tau correct sur le test_data

**Comment by nKachout** [comment]
J'ai fais quelques recherches et j'ai vu que l'utilisation de la fonction d'activation sigmoid ne devait être utiliser que pour les problèmes de classifications car elle ne donne en sortie qu'une valeur comprise entre 0 et 1 qui ne peut donc sur le dernier neurone nous donner q'une valeur maximale de 1 qui ne permet pas d'avoir un temps tau supérieur à 1. J'ai vu qu'il fallait plutôt utiliser la fonction ReLU par exemple qui permet d'avoir un temps tau "valide" mais je n'ai pas encore réussi à changer tout le code de backpropagation pour qu'il fonctionne correctement.

**Comment by nKachout** [comment]
Je crois avoir réussi à "coder" un programme qui fonctionne il reste juste à améliorer les hyper-paramètres pour avoir un écart plus faible que l'écart que j'ai déjà réussi à avoir qui est en moyenne de 0,5. Pour régler le problème j'ai juste attribuer une fonction d'activation sigmoide aux couches cachées et j'ai attribué une fonction d'activation ReLU au neurone de sortie.

**Comment by MLoum** [comment]
Bonjour Najim, 

cela à l'air assez prometteur. Pourrais-tu m'envoyer le code ? 

Je vais avoir deux journées de travail complete mardi et mercredi. Je vais evidemment regarder en détail le projet bibliographique. 

ML 


De: "nKachout" <notifications@github.com> 
À: "MLoum/LifetimeFitNeuralNetwork" <LifetimeFitNeuralNetwork@noreply.github.com> 
Cc: "Subscribed" <subscribed@noreply.github.com> 
Envoyé: Vendredi 22 Mai 2020 21:25:57 
Objet: Re: [MLoum/LifetimeFitNeuralNetwork] évaluation sur le test_data (#1) 





Je crois avoir réussi à "coder" un programme qui fonctionne il reste juste à améliorer les hyper-paramètres pour avoir un écart plus faible que l'écart que j'ai déjà réussi à avoir qui est en moyenne de 0,5. Pour régler le problème j'ai juste attribuer une fonction d'activation sigmoide aux couches cachées et j'ai attribué une fonction d'activation ReLU au neurone de sortie.
